src/AMP.php

```diff
@@ -64,7 +66,7 @@ class AMP
         'Lullabot\AMP\Pass\FacebookNonIframeTransformPass',
         'Lullabot\AMP\Pass\TwitterTransformPass',
         'Lullabot\AMP\Pass\StandardScanPass',
-        'Lullabot\AMP\Pass\StandardFixPass',
+        'Lullabot\AMP\Pass\SugarStandardFixPass',
```

Comment:
Are you really planning to open PR to upstream repo with `Sugar`-prefixed class?